### PR TITLE
Create Reply on Comment

### PR DESCRIPTION
**Pull request by jinchen1036**
### Overview of PR

#102 

### Technical Changes

- New Package `pg-hstore`
  - Reference: https://github.com/scarney81/pg-hstore
- Create` {{unifree}}/api/v1/post/postreply` Post EndPoint
- Example API CALL:
Screen Shot 2019-12-06 at 10 57 07 PM

- Example Get Comment after create reply:
Screen Shot 2019-12-06 at 10 56 55 PM

### CODE REVIEW

File: src/services/discussionService.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
-import { findComments, createComment, deleteComment } from './commentService';
+
 import { findUserbyID } from './userService';
+import {
+  findComments, createComment, deleteComment, updateReply,
+} from './commentService';
 
+const hstore = require('pg-hstore')();
```
Comment (songrenzhao) [inline src/services/discussionService.js]: Any reason this cannot be done in es6 import?
```js
import hstore from 'pg-hstore';
```
Then you can do 
```js
const { stringify } = hstore();
```

File: src/services/discussionService.js [hunk 2]
```diff
@@ -70,3 +74,20 @@ export async function deleteUserComment(postInfo) {
     throw error;
   }
 }
+
+export async function updateUserReply(replyInfo) {
+  try {
+    const { commentID, replyerID, content } = replyInfo;
+    const { username } = await findUserbyID(replyerID);
+    const newReply = hstore.stringify({
```
Comment (songrenzhao) [inline src/services/discussionService.js]: Please replace with 
```js 
stringify({...})
```

File: src/services/discussionService.js (hunk 1 above)
Comment (jinchen1036) [reply src/services/discussionService.js]: fixed

File: src/services/discussionService.js (hunk 2 above)
Comment (jinchen1036) [reply src/services/discussionService.js]: fixed

File: src/services/discussionService.js [hunk 3]
```diff
@@ -70,3 +74,20 @@ export async function deleteUserComment(postInfo) {
...
+export async function updateUserReply(replyInfo) {
+  try {
+    const { commentID, replyerID, content } = replyInfo;
+    const { username } = await findUserbyID(replyerID);
+    const newReply = stringify({
+      content,
+      username,
+      createdAt: (new Date()).toISOString(),
+    });
+
```
Comment (songrenzhao) [inline src/services/discussionService.js]: Remove line

**Pull request merged**